v2/http-middleware/security.go

```diff
@@ -235,8 +232,8 @@ func AuthorizeMiddleware(authorizer Authorizer) mux.MiddlewareFunc {
 				return
 			}
 			if !isAuthorized {
-				responseBody := GetUnauthorizedErrorResponseBody(http_model.ErrResponseUnauthorized, secConfig)
-				http_server.WriteJSONResponse(ctx, w, req, http.StatusUnauthorized, responseBody)
+				responseBody := GetUnauthorizedErrorResponseBody(http_model.ErrResponseForbidden, secConfig)
```

Comment:
This is a breaking change that will be hard to release. Since it will lock go-utility version for all current users.

I think it is the right way forward and therefore we changed it in the new middlewares 
https://github.com/SKF/go-enlight-middleware/